src/main/java/com/automation/school/ad_main/Collection/User.java

```diff
@@ -0,0 +1,20 @@
+package com.automation.school.ad_main.Collection;
+
+import java.util.Map;
+
+public class User {
+
+    public long id;
+    String username = null;
+    public Map<String,Address> addresses; //(i.e. Key= "Home", Value = Address)
+
+
+    public User() {
+        this.username = username;
```

Comment:
nothing would happen

Reply:
Modified on 
 public User() {
        userCurrentId++;
        this.addresses = new HashMap<>();
    }